### PR TITLE
🔨 fix the mdim edit hack

### PR DESCRIPTION
**Pull request by danyx23**
Fixes the hack to make the internal edit link in the share menu of grapher work for Mdims (#3987)

### CODE REVIEW

**Comment by danyx23** [comment]
* **#4105** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/4105?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4104** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/4104?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @danyx23 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: site/multiDim/MultiDimDataPageContent.tsx [hunk 1]
```diff
@@ -278,17 +278,27 @@ export const MultiDimDataPageContent = ({
         }
 
         if (!grapherConfigIsReady) return baseConfig
+        const variables = currentView?.indicators?.["y"]
+        const editUrl =
+            variables?.length === 1
+                ? `variables/${variables[0]}/config`
+                : undefined
```
Comment (marcelgerber) [inline site/multiDim/MultiDimDataPageContent.tsx:285]: Hm, this will mean that if there is more than a single y variable, the URL will _still_ be wrong...

Comment (danyx23) [reply site/multiDim/MultiDimDataPageContent.tsx:285]: Yeah - do you think it's worthwhile turning editUrl into an object so we can communicate "do not show any edit link"?

**Comment by danyx23** [comment]
### Merge activity

* **Nov 14, 12:59 PM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/4104).

**Pull request merged**